### ISSUE TITLE
Intuitive connection between logistic regression and neural networks

### ISSUE THREAD

**Issue by agitter** [gitter-lab/ml-bio-workshop#113]
I have previously used the TensorFlow Playground to explain the intuition behind logistic regression and neural networks without diving in the underlying math.  The website lets the user train models on simple datasets and also interact with the trained model to see how manually adjusting the weights affects the decision boundary.

A few examples I use:
- [Linear decision boundary and linearly-separable data](http://playground.tensorflow.org/#activation=relu&batchSize=10&dataset=gauss&regDataset=reg-plane&learningRate=0.03&regularizationRate=0&noise=0&networkShape=1&seed=0.97800&showTestData=false&discretize=false&percTrainData=50&x=true&y=true&xTimesY=false&xSquared=false&ySquared=false&cosX=false&sinX=false&cosY=false&sinY=false&collectStats=false&problem=classification&initZero=false&hideText=false)
- [Linear decision boundary and non-linearly-separable data](http://playground.tensorflow.org/#activation=relu&batchSize=10&dataset=xor&regDataset=reg-plane&learningRate=0.03&regularizationRate=0&noise=0&networkShape=1&seed=0.77646&showTestData=false&discretize=false&percTrainData=50&x=true&y=true&xTimesY=false&xSquared=false&ySquared=false&cosX=false&sinX=false&cosY=false&sinY=false&collectStats=false&problem=classification&initZero=false&hideText=false)
- [Non-linear decision boundary and complex non-linearly-separable data](http://playground.tensorflow.org/#activation=relu&batchSize=5&dataset=spiral&regDataset=reg-plane&learningRate=0.01&regularizationRate=0&noise=0&networkShape=8,8,8,8&seed=0.94235&showTestData=false&discretize=false&percTrainData=50&x=true&y=true&xTimesY=false&xSquared=false&ySquared=false&cosX=false&sinX=false&cosY=false&sinY=false&collectStats=false&problem=classification&initZero=false&hideText=false)
